internal/pkg/handler/update.go

```diff
@@ -120,13 +117,13 @@ func PerformRollingUpgrade(client kubernetes.Interface, config util.Config, enva
 				if value == config.ResourceName {
 					updated := updateContainers(containers, value, config.SHAValue, envarPostfix)
 					if !updated {
-						logrus.Warnf("Rolling upgrade did not happen")
+						logrus.Warnf("Rolling upgrade did not happen for %s of type %s in namespace: %s", resourceName, upgradeFuncs.ResourceType, config.Namespace)
```

Comment:
why didn't it happen? shouldn't we add the reason to make the log more meaningful?